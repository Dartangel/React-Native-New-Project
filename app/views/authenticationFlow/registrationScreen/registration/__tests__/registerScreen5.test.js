import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Registration from '../registration';

describe('Registration', () => {
    test('Registration snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Registration />);
        expect(snapshot).toMatchSnapshot();
    });
});