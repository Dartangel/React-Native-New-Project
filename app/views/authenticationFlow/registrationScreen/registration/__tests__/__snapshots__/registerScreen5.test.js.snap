// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Registration snapshot 1`] = `
<KeyboardAvoidingView
  behavior="padding"
  enabled={true}
  keyboardShouldPersistTaps="always"
  keyboardVerticalOffset="-250"
>
  <ComponentWrapper>
    <ImageBackground
      source={
        Object {
          "testUri": "../../../app/src/img/splashScreen1.jpeg",
        }
      }
      style={
        Object {
          "height": 1310,
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "alignSelf": "stretch",
            "flex": 1,
            "justifyContent": "center",
            "paddingLeft": 60,
            "paddingRight": 60,
          }
        }
      >
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontSize": 35,
              "paddingBottom": 10,
            }
          }
        >
          Registration
        </Text>
        <View>
          <TextInput
            allowFontScaling={true}
            onChangeText={[Function]}
            placeholder="Username"
            placeholderTextColor="white"
            rejectResponderTermination={true}
            secureTextEntry={false}
            style={
              Object {
                "alignSelf": "stretch",
                "backgroundColor": "rgba(0, 72, 82, 0.6)",
                "borderBottomColor": "#ffffff",
                "borderBottomWidth": 1,
                "borderRadius": 25,
                "color": "#08DBA5",
                "fontSize": 20,
                "height": 60,
                "marginBottom": 10,
                "paddingHorizontal": 10,
              }
            }
            underlineColorAndroid="transparent"
          />
        </View>
        <View>
          <TextInput
            allowFontScaling={true}
            onChangeText={[Function]}
            placeholder="Password"
            placeholderTextColor="white"
            rejectResponderTermination={true}
            secureTextEntry={true}
            style={
              Object {
                "alignSelf": "stretch",
                "backgroundColor": "rgba(0, 72, 82, 0.6)",
                "borderBottomColor": "#ffffff",
                "borderBottomWidth": 1,
                "borderRadius": 25,
                "color": "#08DBA5",
                "fontSize": 20,
                "height": 60,
                "marginBottom": 10,
                "paddingHorizontal": 10,
              }
            }
            underlineColorAndroid="transparent"
          />
        </View>
        <View>
          <TextInput
            allowFontScaling={true}
            onChangeText={[Function]}
            placeholder="Confirm password"
            placeholderTextColor="white"
            rejectResponderTermination={true}
            secureTextEntry={true}
            style={
              Object {
                "alignSelf": "stretch",
                "backgroundColor": "rgba(0, 72, 82, 0.6)",
                "borderBottomColor": "#ffffff",
                "borderBottomWidth": 1,
                "borderRadius": 25,
                "color": "#08DBA5",
                "fontSize": 20,
                "height": 60,
                "marginBottom": 10,
                "paddingHorizontal": 10,
              }
            }
            underlineColorAndroid="transparent"
          />
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
            }
          }
        >
          <NextButton
            onPress={[Function]}
            title="Next"
          />
        </View>
      </View>
    </ImageBackground>
  </ComponentWrapper>
</KeyboardAvoidingView>
`;
